global/global_player.lua

```diff
@@ -231,15 +231,23 @@ vet_aa = {
     [600]  = {441504000, true, true}, -- Blessing of the Devoted 14 yr
 }
 
+
 function event_connect(e)
+	grant_veteran_aa(e)
+    don.fix_invalid_faction_state(e.self)
```

Comment:
formatting, can we do tabs or spaces but not mix (I know we don't have a fixed standard yet, but I will push up a .editorconfig tonight/tomorrow)

Reply:
If you can configure a vscode settings as well, I'm not sure I load .editorconfig with it...

Reply:
@xackery Is this just waiting for a tab?